project/plugins.sbt

```diff
@@ -1,24 +1,17 @@
-import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play26
-
-resolvers ++= Seq(
-  Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(
-    Resolver.ivyStylePatterns),
-  "HMRC Releases" at "https://dl.bintray.com/hmrc/releases",
-  "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
+resolvers += Resolver.bintrayIvyRepo("hmrc", "sbt-plugin-releases")
+resolvers += Resolver.bintrayRepo("hmrc", "releases")
+
+addSbtPlugin(
+  sys.env.get("PLAY_VERSION") match {
+    case Some("2.6") => "com.typesafe.play" % "sbt-plugin" % "2.6.25"
+    case Some("2.7") => "com.typesafe.play" % "sbt-plugin" % "2.7.9"
+    case _           => "com.typesafe.play" % "sbt-plugin" % "2.8.7"
```

Comment:
the default here doesn't match the defaultPlayVersion in PlayCrossCompilation.

Reply:
done.